Content: Cats Watch Fox Hunt on Plains in Winter

Artyphex opened the pull request:
Plain cats watch fox hunting in the snow and either copy it, laugh at it, or snap at each other from the distraction. PRed from its own branch in hopes to avoid confusion in commits

Artyphex committed:
Content: Cats Watch Fox Hunt on Plains in Winter

Plain cats watch fox hunting in the snow and either copy it, laugh at it, or snap at each other from the distraction. PRed from its own branch in hopes to avoid confusion in commits

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
+                "all apprentices": [3, 6]
+            } 
+        },
+        {"weight": 20,
```
theres some { }, brackets in here in teh wrong place, need to be removed - check patrol template

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
             }
         ]
+    },
+    [
```
what's this square bracket doing here?

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
+    },
+    [
+        {
+            "patrol_id": "pln_hunt_watchandlearnfox1",
```
ID needs to include season

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
+                "hunting",
+                "respect",
+                "trust",
+                "patrol_to_s_c",
```
unnecessary tags (we used so specify relationship changes with these tags, but since the patrol code update now we use the relationship code block in the outcomes to do it instead. Same fo rhte hunting tag, it's also not doing anything here)

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
+            "min_max_status": {
+                "deputy": [3, 6],
+                "warrior": [3, 6],
+                "leader": [3,6],
```
can't have 3 leaders

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
+            "min_cats": 3,
+            "max_cats": 6,
+            "min_max_status": {
+                "deputy": [3, 6],
```
can't have 3 deputies

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
+            "relationship_constraint": null,
+            "pl_skill_constraint": null,
+            "intro_text": "Leaf-bare is taking its toll. Your patrol is cold, frustrated, and most of all, hungry. s_c notices something odd. Is that a fox in the distance? Why is it jumping into the snow like that?",
+            "decline_text": "First the cold, and the snow, and now a fox? This is too much for one day, p_l decides. Back to camp, another patrol can try again later. ",
```
extra space at the end of the text

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
+                            "cats_from": ["patrol"],
+                            "mutual": true,
+                            "values": ["platonic"],
+                            "amount": 15
```
slightly too high of a reward for a hunting patrol. between 5-10 increase, pls

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3782,5 +3782,87 @@
+                ]
+                    } 
+                
+    ]
```
I have no idea what's going on with your brackets here but pls fix

Artyphex replied:
I have no explanation for you but I will fix it

Tiritirimatangi reviewed (changes requested):
A lovely atmospheric patrol that'll be ideal for expanding our plains patrols int eh winter! Thank you so much! I've requested the changes needed before this can be merged into dev <3

Artyphex committed:
Content: Fox Winter Plain Hunt - Update

Removed unnecessary tags, added season to ID, fixed min/max cats number, (hopefully) fixed the bracket madness.

Tiritirimatangi commented on resources/dicts/patrols/plains/hunting/leaf-bare.json:
```diff
@@ -3783,86 +3783,74 @@
+                }
             ]
         }
     ]
```
what's this square bracket doing down here?

Artyphex replied:
@Tiritirimatangi  Vstudio is insisting that be there or the whole thing errors out
